.github/workflows/continuous-integration-workflow.yml

```diff
@@ -34,18 +34,45 @@ jobs:
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }} 
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: Convert build_args
         id: build_args
         run: |
-          echo "args<<EOF" >> $GITHUB_OUTPUT 
+          echo "args<<EOF" >> $GITHUB_OUTPUT
           echo ${{ matrix.config.build_args }} | sed 's/,/\n/' >> $GITHUB_OUTPUT
           echo "EOF" >> $GITHUB_OUTPUT
-      - name: Build and Push Docker images for Github Container Registry
-        uses: docker/build-push-action@v3        
+      - name: Build new image and Load it into Docker for testing
+        uses: docker/build-push-action@v4
         with:
-          tags: ghcr.io/kokkos/ci-containers/${{ matrix.config.dockerfile }}:${{ matrix.config.tag }}
+          tags: ${{ env.docker-tag }}
           file: ${{ matrix.config.dockerfile }}
           build-args: ${{ steps.build_args.outputs.args }}
           pull: true
-          push: ${{ github.repository_owner == 'kokkos' && ( github.event_name == 'push' ||  github.event_name == 'schedule' ) }}
```

Comment:
I would really like a `push: false` here instead of relying on the default.

Reply:
Good point. Done.